filament/src/driver/opengl/ContextManagerEGL.cpp

```diff
@@ -537,6 +537,7 @@ std::unique_ptr<Driver> ContextManagerEGL::createDriver(void* const sharedGLCont
             EGL_GREEN_SIZE,  8,
             EGL_BLUE_SIZE,   8,
             EGL_ALPHA_SIZE,  0, // reserved to set ALPHA_SIZE below
+            EGL_RECORDABLE_ANDROID, 1,
```

Comment:
That's okay, but if we can't find a config with RECORDABLE_ANDROID, then we need to try again without it (and maybe print a warning in the log).

Reply:
done.